openff/toolkit/utils/toolkits.py

```diff
@@ -5930,3 +174,50 @@ def __repr__(self):
                 wrapper._toolkit_name, wrapper._toolkit_installation_instructions
             )
     print(msg)
+
+
+# Consistency check that the __all__ is correct.
+def _check_all():
```

Comment:
(blocking) I'm not sure if this was accidentally committed or not.  Right now it makes it so that, if a developer added a new Exception type to `exceptions.py` and updated that file's `__all__`, they'd get an error unless they also added the new Exception to _this_ file's `__all__`. Is that the intent? 

If it's unintentional, I'm ok with this check being deleted

If it's intentional:

* Is it worth having this check at all / do we WANT to force this file's `__all__` to be a superset of all the imported files `__all__`s? 
* Would be be better (or possible) to put this in `test_toolkits` instead of here?
* If this is the pattern we want, would this be generally satisfiable by using `from exceptions import *`?